clients/venice-push-job/src/test/java/com/linkedin/venice/hadoop/TestValidateSchemaAndBuildDictMapperOutputReader.java

```diff
@@ -98,7 +122,8 @@ public void testGetWithNoZstdDictionary() throws Exception {
         1,
         null,
         fileSchema.toString());
-    outputReader = new ValidateSchemaAndBuildDictMapperOutputReader(inputDir.getAbsolutePath() + '/' + avroOutputFile);
-    outputReader.getData();
+    ValidateSchemaAndBuildDictMapperOutputReader reader =
+        new ValidateSchemaAndBuildDictMapperOutputReader(inputDir.getAbsolutePath(), avroOutputFile);
+    reader.close();
```

Comment:
Can we add some validations that the file size field is set as expected and the dict is empty